src/Uno.UWP/ApplicationModel/Contacts/Contact.cs

```diff
@@ -10,26 +10,36 @@
 using Uno.Extensions;
 
 namespace Windows.ApplicationModel.Contacts
-{
+{
 	/// <summary>
 	/// Represents a contact.
 	/// </summary>
 	public partial class Contact
 	{
 		[NotImplemented]
-		public string? Id { get; set; }
-
+		public string? Id { get; set; }
+
+		private void LogWarningLenUWP(string value, int lenLimit, string variableName)

```

Comment:
It's best to use full words:

```suggestion
		private void ValidateStringLength(string value, int maxLength, string variableName)

```